package.json

```diff
@@ -4,8 +4,10 @@
   "description": "Discord bot for Eddie Jaoude's Discord server",
   "main": "index.js",
   "scripts": {
-    "lint": "tslint --project .",
+    "lint": "eslint . --ext .ts",
```

Comment:
Should we add this to the GitHub Action so that it fails the build if something is not correct?